### PR TITLE
Iss1105 - Misc user feedback

### PR DESCRIPTION
**Pull request by jvwong**
refs #1105 

- interaction listing and title 
- interaction tooltip link outs

screenshot at oct 03 10-30-52
screenshot at oct 03 10-31-20
screenshot at oct 03 10-31-35

### CODE REVIEW

File: src/client/features/interactions/index.js [hunk 1]
```diff
@@ -72,7 +73,8 @@ class Interactions extends React.Component {
     let appBar = h('div.app-bar.interactions-bar', [
       h('div.app-bar-branding', [
         h(PcLogoLink),
-        h('div.app-bar-title', sources.join(', ') + ' Interactions')
+        sources.length === 1 ?  h('div.app-bar-title', `Top ${config.MAX_SIF_NODES} interactions involving ${sources[0]}`):
```
Comment (d2fong) [inline src/client/features/interactions/index.js]: should we always display max sif nodes config value if there are less nodes than that?  e.g. top 25 interactions between mdm2 and tp53 when there are only two actual nodes?

**Comment by maxkfranz** [comment]
It's top 25 genes rather than interactions, right?

How about something like this?

n = 1 : Interactions with TP53 and the top 25 genes

For n = 1, we can have up to 25 resultant genes.

n >= 2 : Interactions between TP53, MDM2, and PCNA

For n >= 2, we only ever return the n specified with interactions between them.

**Comment by maxkfranz** [comment]
An alternative for n = 1 : TP53 and its interactions with the top 25 genes

**Comment by jvwong** [comment]
This one has:

n = 1: 'Interactions between TP53 and top 25 genes'
n >= 2: 'Interactions between TP53, MDM2 and CDKN1A'

**Comment by maxkfranz** [comment]
@jvwong @d2fong I've added some padding to make the link sizes a bit bigger, like they were when they had parentheses.  This makes them easier to click.

**Comment by d2fong** [comment]
You are going to have to cherry pick all of these commits to v2.0.0.  Thats the release branch.

**Comment by d2fong** [comment]
@maxkfranz  @jvwong 

Also don't merge development into v2.0.0.  There is stuff in development that shouldn't go into v2.0.0.

**Comment by maxkfranz** [comment]
Cherry-picked this PR into v2.0.0

**Pull request merged**